### DISCUSSION TITLE
Expression of Interest: Contributing to Openpedia

### DISCUSSION THREAD

**Discussion by htetaunglin-coder** [Sriparno08/Openpedia#70] [Q&A] [answered]
Hi @Sriparno08,

I'm reaching out to let you know that I'm interested in contributing to this open-source project. If there are specific areas or tasks that you'd like assistance with, or if there are issues labeled for newcomers, I'd love to take on a task and make a meaningful contribution to the project.

Could you provide some guidance on how I can get started? Are there particular areas that need attention, or any specific tasks you'd recommend for someone new to the project?

Thank you for your time, and I'm looking forward to the opportunity to contribute.

Best regards,
HTLA

**Comment by Sriparno08** [comment, marked as answer]
Hey @HTLA380, thanks for your interest!

To contribute to this project, you can go to the 'Issues' tab and take a look at the open issues. Currently, all the open issues in the tab are suitable for beginners. Take a look at them and see if you can contribute to any of them.

If you find one, comment on the issue that you want to work on it. You'll be assigned to it and hence, become eligible to start working on it.

Before contributing, please go through the [CONTRIBUTING](https://github.com/Sriparno08/Openpedia/blob/main/CONTRIBUTING.md) guide for a list of guidelines that you need to follow in order to contribute to this project.

Good luck!

**Reply by htetaunglin-coder** [reply]
Hello @Sriparno08 
I hope you're doing well! 👋 I noticed that your live site is currently under construction, and I'd be thrilled to lend a hand in building it. If you're open to it, I can assist with coding and implementing any design files you have in mind. Let me know how I can contribute to bringing your site to life!

**Comment by Sriparno08** [comment]
Hey @HTLA380, thanks for the helping hand. We're currently planning the website structure that we want to implement. Once it is finalized, we'll implement it by opening up a handful of beginner-friendly issues that you can contribute to. So, keep an eye on the repository for future issues.

For now, you can contribute to the issues that are currently open. Speaking of that, I can see your comment on one of those issues. So, let me assign it to you so you can start your work.

P.S.: You've slightly messed up the person that you are saying hello to, since I was the one who created this site! 🙂
